azure-quantum/tests/unit/test_solvers.py

```diff
@@ -44,3 +44,24 @@ def test_number_of_solutions_set(testsolver):
     testsolver.set_number_of_solutions(100)
     assert param_name in testsolver.params["params"]
     assert testsolver.params["params"][param_name] == 100
+
+def test_submit_proto_problem(self):
```

Comment:
is this a live test? we should add the @pytest.live mark and probably the @pytest.qio mark.
If it needs a special region and/or provider the capability that controls the test might be different.
If this is not a live test, we should add a couple.